### PR TITLE
Option to insert badges

### PR DESCRIPTION
**Pull request by andrelmlins**
* Goal to enter the option to list project badges in foot.
* Created the configuration badges (bool).
* Removing console.log.

### CODE REVIEW

File: src/components/project-card.vue [hunk 1]
```diff
@@ -51,6 +56,7 @@
       pipelines: null,
       pipelineCount: 0,
       refNames: [],
+      badges:[],
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
      badges: [],
```

File: src/components/project-card.vue [hunk 2]
```diff
@@ -91,7 +97,8 @@
     },
     watch: {
       project() {
-        this.fetchPipelines()
+        this.fetchPipelines();
+        if(Config.root.badges) this.fetchBadges();
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
        if (Config.root.badges) this.fetchBadges()
```

File: src/components/project-card.vue [hunk 3]
```diff
@@ -91,7 +97,8 @@
     },
     watch: {
       project() {
-        this.fetchPipelines()
+        this.fetchPipelines();
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
        this.fetchPipelines()
```

File: src/components/project-card.vue [hunk 4]
```diff
@@ -243,6 +250,10 @@
         this.refNames = refNames
         this.pipelineCount = count
         this.loading = false
+      },
+      async fetchBadges() {
+        const badges = await this.$api(`/projects/${this.projectId}/badges`);
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
        const badges = await this.$api(`/projects/${this.projectId}/badges`)
```

File: src/components/project-card.vue [hunk 5]
```diff
@@ -243,6 +250,10 @@
...
+        this.badges = badges;
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
        this.badges = badges
```

File: src/components/project-card.vue [hunk 6]
```diff
@@ -21,6 +21,11 @@
     </div>
     <div class="spacer"></div>
     <div class="info">
+      <div class="container-badges">
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
      <div class="badge-container">
```

File: src/components/project-card.vue [hunk 7]
```diff
@@ -329,5 +340,22 @@
         margin-right: 4px;
       }
     }
+
+    .container-badges {
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
    .badge-container {
```

File: src/components/project-card.vue [hunk 8]
```diff
@@ -329,5 +340,22 @@
...
+      max-width:80%;
+    }
+
+    .badge {
```
Comment (timoschwarzer) [inline src/components/project-card.vue]: Please move this block inside `.badge-container`

File: src/components/project-card.vue [hunk 9]
```diff
@@ -329,5 +340,22 @@
...
+      margin-right: 8px;
+        transition: background-color 100ms linear, color 100ms linear, border 100ms linear;
+      
+      img {
+        opacity: 0.9;
+      }
+
+      img:hover {
```
Comment (timoschwarzer) [inline src/components/project-card.vue]: Please use `&:hover { ... }` inside the `img` block here

**Comment by andrelmlins** [comment]
Examples:

image

The projects you are using are internal to a business and can not be submitted, I have edited the content.

**Comment by andrelmlins** [comment]
Another Example:

image

File: src/components/project-card.vue [hunk 10]
```diff
@@ -329,5 +340,22 @@
         margin-right: 4px;
       }
     }
+
+    .badge-container {
+      max-width:80%;
+
+      a {
+        margin-right: 8px;
+          transition: background-color 100ms linear, color 100ms linear, border 100ms linear;
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
        transition: background-color 100ms, color 100ms, border 100ms;
```

File: src/components/project-card.vue [hunk 11]
```diff
@@ -92,6 +98,7 @@
     watch: {
       project() {
         this.fetchPipelines()
+        if(Config.root.badges) this.fetchBadges()
```
Comment (timoschwarzer) [inline src/components/project-card.vue]:
```suggestion
        if (Config.root.badges) this.fetchBadges()
```

**Comment by timoschwarzer** [comment]
Some small things left, then we should be good to go :)

**Comment by andrelmlins** [comment]
Done

**Comment by timoschwarzer** [comment]
Thank you! :)

**Comment by timoschwarzer** [comment]
fyi, I disabled the badges by default.

**Comment by andrelmlins** [comment]
Oh, sorry, I was interested in leaving him disabled, I did not remember at the moment

**Pull request merged**